modules/social_features/social_landing_page/social_landing_page.module

```diff
@@ -299,6 +299,10 @@ function _social_landing_page_get_hero_image(NodeInterface $node) {
 
   /** @var \Drupal\paragraphs\ParagraphInterface $paragraph_section */
   foreach ($landing_page_section_field->referencedEntities() as $paragraph_section) {
+    if ($paragraph_section->get('field_section_paragraph')->isEmpty()) {
```

Comment:
I will recommend making it a "positive" case condition, i.e, something like:

`if ($paragraph_section->get('field_section_paragraph')->isEmpty() !== TRUE) {`